app/serializers/service_provider_serializer.rb

```diff
@@ -9,6 +9,7 @@ class ServiceProviderSerializer < ActiveModel::Serializer
     :cert,
     :friendly_name,
     :ial,
+    :default_aal,
```

Comment:
can we add specs for this class too? or the controller that serves this serializer?

Reply:
Are you looking for more than this?
https://github.com/18F/identity-dashboard/pull/388/files#diff-b698ab666b7c150a9fb27a9931adbf4ca9e203ae16ac80a728fcf1d24dbff068R29-R30

Reply:
Nope! I must have missed that, thanks!